docroot/sites/its.uiowa.edu/modules/its_core/src/Plugin/Block/AlertTypeLegend.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\its_core\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+
+/**
+ * Provides an 'Alert Type Legend' block.
+ *
+ * @Block(
+ *   id = "alert_type_legend_block",
+ *   admin_label = @Translation("Alert Type Legend"),
```

Comment:
```suggestion
 *   admin_label = @Translation("Alert type legend"),
```
I know we have slipped previously but can we adjust the capitalization on this before it is sent out?

<img width="275" alt="Screenshot 2024-08-12 at 10 41 41 AM" src="https://github.com/user-attachments/assets/e54420e5-36e3-4f99-80b8-7a249ebdd0e2">